--- a/drivers/bluetooth/btusb.c 2015-10-28 14:52:20.466644715 +0000
+++ b/drivers/bluetooth/btusb.c 2015-10-28 14:52:34.715107143 +0000
@@ -2657,7 +2657,7 @@
 	BT_DBG("intf %p id %p", intf, id);
 
 	/* interface numbers are hardcoded in the spec */
-	if (intf->cur_altsetting->desc.bInterfaceNumber != 0)
+	if (intf->cur_altsetting->desc.bInterfaceNumber != 2)
 		return -ENODEV;
 
 	if (!id->driver_info) {
@@ -2827,7 +2827,7 @@
 		data->isoc = NULL;
 	} else {
 		/* Interface numbers are hardcoded in the specification */
-		data->isoc = usb_ifnum_to_if(data->udev, 1);
+		data->isoc = usb_ifnum_to_if(data->udev, 3);
 	}
 
 	if (!reset)

